tests/e2e/default/navigation/more-options-menu/online-user.wdio-spec.js

```diff
@@ -0,0 +1,81 @@
+const commonPage = require('../../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../../page-objects/default/login/login.wdio.page');
+const contactPage = require('../../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../../page-objects/default/reports/reports.wdio.page');
+const seeder = require('./seeder');
+
+describe('Export tests', async () => {
+  before(async () => {
+    await loginPage.cookieLogin();
+  });
+
+  describe('Export disabled when no items: messages, contacts, people', async () => {
+    //empty db
+    it('- Message tab', async () => {
```

Comment:
The test case description should be clear and descriptive, for example:

`should disable the Export option in menu when user has permissions`

Please apply to the other test cases below. Thanks!